packages/terra-action-footer/src/CenteredActionFooter.jsx

```diff
@@ -12,27 +12,37 @@ const propTypes = {
    * Actions to be displayed in the center socket
    */
   center: PropTypes.node,
+  /**
+   * Whether or not the center socket should display as a block.
+   */
+  isBlock: PropTypes.bool,
 };
 
 const defaultProps = {
   center: undefined,
+  isBlock: false,
 };
 
 const CenteredActionFooter = ({
   center,
+  isBlock,
   ...customProps
 }) => {
   const centeredActionFooterClassNames = cx([
     'centered-action-footer',
     customProps.className,
   ]);
 
+  const isBlockClassName = cx([
+    { 'is-block': isBlock },
+  ]);
+
   return (
     <ActionFooterContainer
       {...customProps}
       className={centeredActionFooterClassNames}
     >
-      {center}
+      {center && cloneElement(center, { className: isBlockClassName })}
```

Comment:
Wouldn't this override custom classes applied to center?

Reply:
It seems like we almost want a new variant of ActionFooter ('BlockButtonFooter' or something less awful) that simply takes an array of Buttons. That footer could own and style container as it needs to without bleeding styles into consumer-provided components.

Reply:
I totally missed @bjankord's suggestion in Teams. Thoughts on this instead of what I currently have?
<img width="438" alt="screen shot 2018-06-26 at 9 38 20 am" src="https://user-images.githubusercontent.com/34115417/41919686-b59609d2-7924-11e8-9854-8bf5f41c4a98.png">

Reply:
What styles would the `isBlock` prop be changing on the CenteredActionFooter? 

I could see having a `layoutDirection` prop (with `horizontal` or `vertical` values) that would change the flex-direction accordingly. With it set to vertical, an array of block Buttons could be rendered without the need of a wrapper.

We could also play with the align-items value and change to to `strech` when vertical to prevent the need for `isBlock` on the Buttons, but I'm not sure if that behavior is always going to be desired.

Reply:
Here's @bjankord's suggested change: 6395924e497ca9978903e9ff390454fa140dc183

deployed [here](https://terra-core-deployed-pr-1648.herokuapp.com/#/components/terra-action-footer/action-footer/centered)

Reply:
> What styles would the isBlock prop be changing on the CenteredActionFooter?

The display style of the action footer container would be changed from `flex` to `block`.

Reply:
Ah I see the issue with that. I like the idea of having a `layoutDirection` prop to switch `flex-direction`. I do think the stacked button use case is the primary reason the CenteredActionFooter exists. I wouldn't be opposed to setting `align-items: stretch` if `layoutDirection` is set to `vertical`. If it becomes a concern those are coupled, we could split the ability to set `align-items: stretch` to a separate prop.

Reply:
@bjankord so this? One side effect of this is that buttons automatically fill out the center socket. This of course could be alleviated if we decouple the `align-items: stretch;` from `layoutDirection`.
![image](https://user-images.githubusercontent.com/34115417/42042123-0f5ca04e-7ab9-11e8-94dc-4844ba318a62.png)

Reply:
Discussed in a meeting last Friday a path forward for this. Plan is to expose [_ActionFooterContainer.jsx](https://github.com/cerner/terra-core/blob/master/packages/terra-action-footer/src/_ActionFooterContainer.jsx) as a new component named BlockActionFooter.

To update, we can rename `_ActionFooterContainer.jsx` to `BlockActionFooter.jsx`. Then in the other components, update the import of `_ActionFooterContainer.jsx` to `BlockActionFooter.jsx`.